CHANGELOG.rst

```diff
@@ -11,6 +11,12 @@ Change Log
 
 .. There should always be an "Unreleased" section for changes pending release.
 
+[1.2.1] - 2020-12-17
+~~~~~~~~~~~~~~~~~~~~
+
+* Improve monitoring of legacy Waffle class imports. We should watch for "edx_toggles.toggles.waffle.legacy.WaffleSwitch" custom attributes.
```

Comment:
Nit: Technically, the custom attribute name will be the module (without the class), and the legacy module is under internal, but this is the right idea.